src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1322,6 +1322,9 @@
     <Field Name="Expanded" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
     <Field Name="ResultKind" PropertyOverrides="true" Type="LookupResultKind"/>
+
+    <!-- Used by IOperation to reconstruct the receiver for this expression. -->
+    <Field Name="ReceiverSymbol" Type="TypeSymbol" Null="disallow"/>
```

Comment:
Consider `ReceiverType` rather than `ReceiverSymbol`. The latter sounds like the actual receiver (for instance, the `LocalSymbol` being created).

Reply:
> Consider ReceiverType rather than ReceiverSymbol. The latter sounds like the actual receiver (for instance, the LocalSymbol being created).

Or TypeBeingInitialized?